assets/src/application/services/utilities/text/shortenGuid/index.ts

```diff
@@ -1,14 +1,11 @@
-import { EntityId, EntityDbId } from '@appServices/apollo/types';
+import { EntityId } from '@appServices/apollo/types';
 
 /**
- * converts a GUID like "RGF0ZXRpbWU6NQ==" into "WU6NQ=="
+ * converts a GUID like "RGF0ZXRpbWU6NQ==" into "U6NQ=="
  */
-const shortenGuid = (guid: EntityId | EntityDbId, start = 9, end = 16): string | EntityDbId => {
-	if (typeof guid === 'string' && guid.length > start) {
-		// use a smaller more unique portion of the CUID
-		return guid.substring(start, end);
-	}
-	return guid;
+const shortenGuid = <T extends EntityId>(guid: T): string => {
+	// Return last 6 characters
```

Comment:
good job here;
`slice` > `splice`, since it returns a copy, but do not change the content of the original string